src/Tizen.NUI.Components/Controls/ScrollableBase.cs

```diff
@@ -410,15 +384,74 @@ public bool HideScrollBar
             }
         }
 
+        /// <summary>
+        /// Container which has content of ScrollableBase.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public View ContentContainer { get; private set; }
+
+        /// <summary>
+        /// Set the layout on this View. Replaces any existing Layout.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public new LayoutItem Layout
+        {
+            get
+            {
+                return ContentContainer.Layout;
+            }
+            set
+            {
+                ContentContainer.Layout = value;
+                if(ContentContainer.Layout != null)
+                {
+                    ContentContainer.Layout.SetPositionByLayout = false;
+                }
+            }
+        }
+
+        /// <summary>
+        /// List of children of Container.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public new List<View> Children
+        {
+            get
+            {
+                return ContentContainer.Children;
+            }
+        }
+
+        /// <summary>
+        /// Deceleration rate of scrolling by finger.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float DecelerationRate
+        {
+            get
+            {
+                return decelerationRate;
+            }
+            set
+            {
+                decelerationRate = value;
+                logValueOfDeceleration = (float)Math.Log(value);
+            }
+        }
+
+        /// <summary>
+        /// Threashold not to go infinit.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected float ScrollAnimationThreshold{get;set;} = 0.01f;
```

Comment:
Please use auto format code in visual studio(Ctrl + K + F)

Reply:
Done